### ISSUE TITLE
Accessibility information

### ISSUE THREAD

**Issue by vvo** [tbleckert/react-select-search#93]
Hi there, thanks for this library, it has the good defaults I would expect from a nice combobox.

I have one question: how accessible is the library really? I am only asking because all other libraries like downshift and react-select are heavy on all the "aria*" properties, while there's none in your module.

I am mostly being curious, I have 0 accessibility knowledge and would like more information from you, thanks!

**Comment by tbleckert** [comment]
Great question. It needs some improvements, obviously (as with react-select that uses multiple aria attributes in the wrong way). That being said we have semantic markup, keyboard support and an aria role for the "menu items". Note that the `role` attribute as also WAI related, not only aria-*.

aria-* attributes is a way to tell a screenreader what it is or the purpose of the element when the markup itself isn't enough. What we currently is missing is the `aria-expanded` attribute to tell the screenreader when the select box is open and not.
